Makefile

```diff
@@ -1,8 +1,12 @@
 prepare:
+	rustup default nightly-2021-09-17-x86_64-unknown-linux-gnu
```

Comment:
This line breaks Rust the installation on a Mac. After running the make commands, I had to reinstall Rust from scratch in order to get NCTL to work again.